### DISCUSSION TITLE
What is the difference between antd and @formily/antd ?

### DISCUSSION THREAD

**Discussion by KrishnaPG** [alibaba/formily#3554] [Q&A 2.x] [answered]
In some examples of the docs, components from `antd` are used directly and in some places we see `@formily/antd` components being used. Which one is recommended and what are the differences? For example: 

  - [here](https://react.formilyjs.org/api/components/schema-field) is an example documentation that uses `antd` components
```js
import React from 'react'
import { createForm } from '@formily/core'
import { FormProvider, createSchemaField } from '@formily/react'
import { Input, Select } from 'antd'  /// <<<---
```
  - [here](https://antd.formilyjs.org/components/input) is similar example but uses `@formily/antd`
 
```js
import React from 'react'
import { createForm } from '@formily/core'
import { FormProvider, createSchemaField } from '@formily/react'
import { Input, FormItem, FormButtonGroup, Submit } from '@formily/antd'  /// <<<---
``` 

Question: 

Which one of `antd` or `@formily/antd` should be used, since they both seem to have few components such as `Input`, `Checkbox` etc. common
-------------

**Comment by lumdzeehol** [comment, marked as answer]
`@formily/antd` is recommended. ` @formily/antd` is actually base on `antd`, and it was connected with some reactive properties from `field` model.  For example, `<Input >` from `@formily/antd` looks as below, it's connected with the `loading` and `validating` status from `Field`. Once the field is handling some async job like fetching or validating, the origin `antd/Input` will show loading indicator. 
https://github.com/alibaba/formily/blob/48fd1842e051885ff8edb4617c0c92f4931ccdb2/packages/antd/src/input/index.tsx#L12-L29

In short, `@formily/antd` is like a HOC for `antd`, injecting with  field model properties.

**Reply by KrishnaPG** [reply]
Thank you for the detailed explanation @lumdzeehol
